lib/src/bottom_sheets/cupertino_bottom_sheet.dart

```diff
@@ -252,8 +268,11 @@ class CupertinoScaffold extends StatefulWidget {
       context.dependOnInheritedWidgetOfExactType<_CupertinoScaffold>();
 
   final Widget body;
+  final Radius topRadius;
 
-  const CupertinoScaffold({Key key, this.body}) : super(key: key);
+  const CupertinoScaffold(
```

Comment:
I don't know if this is the best way to add topRadius to CupertinoScaffold. Can you see another way that you don’t need to pass the topRadius twice (on constructor and static method)?

Reply:
@rodineijf You can use CupertinoScaffold.of(context). topRadius inside the static method

Reply:
done